### PR TITLE
[IMP] Update README

### PR DESCRIPTION
**Pull request by vehi-invitu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,100 @@
-# env_invitu_dev
-depot pour mettre en place un env de dev 
+# Installation d'un environnement de developpement Odoo
+> env_invitu_dev
+
+## Development environment installation :
+
+### 1. Create repos for your environment
```
Comment (cvinh) [inline README.md]: This is not a repos but a workspace on your local environment
Maybe better to explain like this :
This script has been written to have a quickly odoo environment using python env and git stuff.
The workspace on the developer computer should be like this : 
~/invitu-devel/allrepos : all the repos cloned from github
~/invitu-devel/projects : all the projects the developer works on
For example, we can have 3 repos gitodoo, gitoca, gitinvitu respectively https://github.com/odoo/odoo, https://github.com/oca, https://github.com/invitu
For a project named "test" in Odoo v17, we will have  ~/invitu-devel/projects/test17

File: README.md [hunk 2]
```diff
@@ -1,2 +1,100 @@
...
-depot pour mettre en place un env de dev 
+# Installation d'un environnement de developpement Odoo
+> env_invitu_dev
+
+## Development environment installation :
+
+### 1. Create repos for your environment 
+example : /invitu-devel/allrepos/gitinvitu/
+```
+mkdir [name_repos]
```
Comment (cvinh) [inline README.md]: no need to put the commands mkdir or cd, the reader is supposed to know those commands

File: README.md [hunk 3]
```diff
@@ -1,2 +1,100 @@
...
+cd [name_repos]
+```
+### 2. Clone the dev environment from github :
+```
```
Comment (cvinh) [inline README.md:20]: in ~/invitu-devel/allrepos/gitinvitu, clone this repo

File: README.md [hunk 4]
```diff
@@ -1,2 +1,100 @@
...
+sudo dnf install git gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite sqlite-devel openssl-devel xz xz-devel libffi-devel patch
+```
+2/ Use this command curl to install
+```
+curl https://pyenv.run | bash
+```
+3/ Add this line into your .bash_profile
+```
+> modify (vim)
+vim ~/.bash_profile
```
Comment (cvinh) [inline README.md]: no need to put the command vim, the reader is supposed to know it or can use another editor

File: README.md [hunk 5]
```diff
@@ -1,2 +1,100 @@
...
+
+
+### Launch file : (from your env_invitu_dev repos)
+```
+./create_environment.sh
+```
+#### If pyenv is **not** installed yet, follow the **Pyenv installation above**.
+### Modules :
+
+1. create a 'modules.csv' file in your project repos (/projects/[project_name])
```
Comment (cvinh) [inline README.md]: humm, review please, it's confused

File: README.md [hunk 6]
```diff
@@ -1,2 +1,100 @@
...
+4/ Restart your shell for the changes to take effects
+
+
+### Launch file : (from your env_invitu_dev repos)
+```
+./create_environment.sh
+```
+#### If pyenv is **not** installed yet, follow the **Pyenv installation above**.
+### Modules :
+
```
Comment (cvinh) [inline README.md:72]: explain what is the role of module.csv and explain how to fill it

File: README.md [hunk 7]
```diff
@@ -1,2 +1,100 @@
...
+1. create a 'modules.csv' file in your project repos (/projects/[project_name])
+2. create your module repos (in /allrepos/) and clone github url (only for public github project)
+
+```
```
Comment (cvinh) [inline README.md:78]: it's not about module repos here, it's just where to clone the github repos

File: README.md [hunk 8]
```diff
@@ -1,2 +1,100 @@
...
+
+> postgresql version install :
+dnf install postgresql15-devel
+
+> python version install and requirements :
+dnf install python-ldap
+and launch (python requirements) > $HOME/.pyenv/versions/odoo17/bin/pip3.12 install -r requirements.txt
+
+> openldap install :
+dnf install openldap-devel
```
Comment (cvinh) [inline README.md]: this has to be done before installing requirements
You forgot to add the export PATH for pgsql binaries

File: README.md [hunk 9]
```diff
@@ -1,2 +1,100 @@
...
+pyenv versions | grep -m 1 {pyenvname}
+pyenv virtualenv {pyenv-version} {pyenvname}
+```
+
+#### If pyenv **is** installed :
+```
+./launch_project.sh
+```
+
+### Install postgresql/python/openldap : (in repos odoo)
```
Comment (cvinh) [inline README.md]: maybe it's no use telling how to install postgresql
but it's a good thing to tell how to install prerequisities for Odoo

**Review by cvinh — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -31,69 +38,68 @@ sudo dnf install git gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite sqli
...
+### Launch file : (from your env_invitu_dev repos)
+```
+./create_environment.sh
 ```
+### Modules :
+The module.csv file is where are defining the modules that will be installed in your Odoo environment.
+It is filled with 2 elements : module_name,module_path
+> example : 
+> partner_firstname,gitoca/partner-contact
+
```
Comment (cvinh) [inline README.md:95]: Please add an example with the url of a pending PR

File: README.md (hunk 1 above)
Comment (vehi-invitu) [reply README.md]: Ok

File: README.md (hunk 2 above)
Comment (vehi-invitu) [reply README.md]: Ok

File: README.md (hunk 4 above)
Comment (vehi-invitu) [reply README.md]: Ok

File: README.md (hunk 5 above)
Comment (vehi-invitu) [reply README.md]: Ok

File: README.md (hunk 6 above)
Comment (vehi-invitu) [reply README.md:72]: Ok

File: README.md (hunk 7 above)
Comment (vehi-invitu) [reply README.md:78]: Ok

File: README.md (hunk 10 above)
Comment (vehi-invitu) [reply README.md:84]: Ok

File: README.md (hunk 9 above)
Comment (vehi-invitu) [reply README.md]: Ok

File: README.md (hunk 8 above)
Comment (vehi-invitu) [reply README.md]: Ok

File: README.md (hunk 10 above)
Comment (vehi-invitu) [reply README.md:95]: is it good if I wrote the example like this ? 
2/ module_name,url_branch_PR (access the Pull request(PR) and retrieve the url of the PR source branch)

Comment (cvinh) [reply README.md:95]: I would have said PR_branch_url
And give an example

Comment (vehi-invitu) [reply README.md:95]: OK.
The example is this one : (in modules.csv)
> example : vidal_connector,https://github.com/dixmit/connector-vidal/tree/17.0-fix

**Review by cvinh — APPROVED** [review]

**Pull request merged**